test/e2e/fixtures.go

```diff
@@ -26,9 +27,13 @@ import (
 	"github.com/stretchr/testify/require"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit, let's use the `corev1` alias for this pkg where it's used so we avoid the double import.

Reply:
done